src/cryptoadvance/specter/controller.py

```diff
@@ -69,7 +69,7 @@ def broadcast(wallet_alias):
     if request.method == 'POST':
         d = request.json
         tx = d['tx']
-        if wallet.cli.testmempoolaccept([tx]):
+        if json.loads(wallet.cli.testmempoolaccept([tx]))['allowed']:
```

Comment:
testmempoolaccept takes an array and returs an array

`if json.loads(wallet.cli.testmempoolaccept([tx]))[0]['allowed']:`